### PR TITLE
FEATURE: Add option to leave public events

### PR DESCRIPTION
**Pull request by jancernik**

### CODE REVIEW

File: app/controllers/discourse_post_event/invitees_controller.rb [hunk 1]
```diff
@@ -45,7 +45,7 @@ def create
     def destroy
       event = Event.find_by(id: params[:post_id])
       invitee = event.invitees.find_by(id: params[:id])
-      guardian.ensure_can_act_on_discourse_post_event!(event)
+      guardian.ensure_can_act_on_invitee!(invitee)
```
Comment (pmusaraj) [inline app/controllers/discourse_post_event/invitees_controller.rb:48]: Can we add a test that ensures user can delete their own invitee record? I think the current tests in `invitees_controller_spec.rb` are checking that admin can delete and lurker cannot delete.

**Review by pmusaraj — CHANGES_REQUESTED** [review]
This looks good, but a test is necessary IMO.

File: app/controllers/discourse_post_event/invitees_controller.rb (hunk 1 above)
Comment (jancernik) [reply app/controllers/discourse_post_event/invitees_controller.rb:48]: Sure, I'll add one

**Review by pmusaraj — APPROVED** [review]
Thanks! LGTM

**Pull request merged**